src/Tests/Moryx.Model.Tests/SqliteTests.cs

```diff
@@ -0,0 +1,98 @@
+using Moryx.Runtime.Kernel;
+using Moryx.TestTools.Test.Model;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moryx.Model.Tests
+{
+    [TestFixture]
+    public  class SqliteTests
+    {
+        private DbContextManager _dbContextManager;
+        private DatabaseConfigUpdateService _databaseConfigUpdateService;
+        private string _targetModelTypename;
+        private Type _configuratorType;
+
+        [SetUp]
+        public void Setup()
+        {
+
+            _dbContextManager = new DbContextManager(CreateConfigManager(), new LoggerFactory());
+            _databaseConfigUpdateService = new DatabaseConfigUpdateService(_dbContextManager);
+            _targetModelTypename = typeof(TestModelContext).FullName;
+            _configuratorType = typeof(SqliteModelConfigurator);
+        }
+
+        [Test]
+        public async Task SqliteCreateDatabaseShouldWork()
+        {
+            string databaseName = "TestCreateProductDatabase";
+            string datasource;
+            SqliteDatabaseConfig dbConfig;
+            SqliteModelConfigurator configurator;
+            InitializeSqliteTools(databaseName, out datasource, out dbConfig, out configurator);
+
+            var result = await configurator.TestConnection(dbConfig);
+            Assert.AreEqual(result, TestConnectionResult.ConnectionOkDbDoesNotExist);
+
+            bool isCreated = await configurator.CreateDatabase(dbConfig);
+
+            Assert.IsTrue(isCreated);
+            Assert.IsTrue(File.Exists(datasource));
+
+            //remove the database
+            await configurator.DeleteDatabase(dbConfig);
+        }
+
+
+        [Test]
+        public async Task SqliteDeleteDatabaseShouldWork()
+        {
+            string databaseName = "TestDeleteProductDatabase";
+            string datasource;
+            SqliteDatabaseConfig dbConfig;
+            SqliteModelConfigurator configurator;
+            InitializeSqliteTools(databaseName, out datasource, out dbConfig, out configurator);
+
+
+            var connectionResult = await configurator.TestConnection(dbConfig);
+            Assert.AreEqual(connectionResult, TestConnectionResult.ConnectionOkDbDoesNotExist);
+
+            bool isCreated = await configurator.CreateDatabase(dbConfig);
+            Assert.IsTrue(isCreated);
+
+            await configurator.DeleteDatabase(dbConfig);
+            Assert.IsFalse(File.Exists(datasource));
+        }
+
+        private static ConfigManager CreateConfigManager()
+        {
+            var configManager = new ConfigManager
+            {
+                ConfigDirectory = ""
+            };
+            return configManager;
+        }
+
+        private static void InitializeSqliteTools(string databaseName, out string datasource, out SqliteDatabaseConfig dbConfig, out SqliteModelConfigurator configurator)
```

Comment:
* I'd remove the `out` parameters 
* and make the `configurator` a class member, since it seems to be the 'SUT'